docs/INSTALL.md

```diff
@@ -0,0 +1,23 @@
+# 💻 How to install?
+
+For installing the Fuel Wallet you need to download [FuelWallet zip file](https://fuels-wallet.vercel.app/preview/fuel-wallet.zip) and import directly on your extensions management page, on chrome or brave.
+
+<p align="center">
+<img src="./assets/install-extension.gif" alt="Install instructions" width="100%">
+<br/>
+<a href="https://user-images.githubusercontent.com/3941923/196786368-8accd2a0-e7b6-4ae2-a6e6-dace918b3266.mp4">Open video</a>
+</p>
+
+## Instructions
+
+1. Inside Chrome or Brave
+2. Open the extensions page, it can be done by;
+   2.1 Clicking on settings -> extensions or;
+   2.2 Accessing `brave://extensions/` or `chrome://extensions/`.
+3. Enable the "Developer mode" switch on the top left.
+4. Load `fuel-wallet.zip` by;
+   4.1 Drag your downloaded Fuel Wallet file, and Drop it in the Extensions page or;
+   4.2 Clicking on `Load unpacked` and selecting the file.
+5. If all went right, an onboarding page will instantly open.
+
+⚡️ Fuel Wallet extension is now ready for your best use. ✅
```

Comment:
```suggestion
1. Inside Chrome or Brave
2. Open the extensions page; it can be done by:
   2.1 Clicking on settings -> extensions or;
   2.2 Accessing `brave://extensions/` or `chrome://extensions/`.
3. Enable the "Developer mode" switch on the top left
4. Load `fuel-wallet.zip` by:
   4.1 Dragging your downloaded Fuel wallet file and dropping it in the extensions page or;
   4.2 Clicking on `Load unpacked` and select the file.
5. If all goes right, an onboarding page will instantly open.

⚡️ Fuel Wallet extension is now ready for your best use. ✅
```